### PR TITLE
make rsnap forcing

### PR DESCRIPTION
**Pull request by tinect**

### CODE REVIEW

**Comment by shyim** [comment]
What does forcing?

**Comment by tinect** [comment]
Continue on error, but outputs it :-)

**Comment by shyim** [comment]
make it configureable like swdc rsnap xxx -f

File: modules/base/rsnap.sh [hunk 1]
```diff
@@ -3,11 +3,22 @@
 SNAP_SUFFIX="$3"
 SNAP_DIR="${CODE_DIRECTORY}/snapshots"
 CONTAINER=$4
+args=""
```
Comment (aragon999) [inline modules/base/rsnap.sh:6]: I guess it should be `ARGS` since it is a "global" variable? :)

Comment (tinect) [reply modules/base/rsnap.sh]: You have seen nothing! :-)

File: modules/base/rsnap.sh [hunk 2]
```diff
@@ -3,11 +3,22 @@
...
 
 if [[ -z $CONTAINER ]]; then
     CONTAINER="mysql"
 fi
 
+shift 3
+while (( $# )); do
+    case $1 in
+        --force)
+            shift
```
Comment (shyim) [inline modules/base/rsnap.sh:16]: You dont need it here

**Pull request merged**